lib/api/redfish-1.0/systems.js

```diff
@@ -283,12 +285,32 @@ var getSystem = controller(function(req, res) {
                 return redfish.handleError(error, res);
             });
         } else if(result.vendor === 'Cisco'){
-	     return dataFactory(identifier,'UCS').then(function(data){
-                options.skuInfo = node.sku === undefined ? "": node.sku;
-		 return redfish.render('ucs.1.0.0.computersystem.1.0.0.json',
-				'ComputerSystem.v1_3_0.json#/definitions/ComputerSystem',
-                               _.merge(options, data));
-	    });
+            return Promise.props({
+                hardware: dataFactory(identifier,'UCS'),
+                bios: dataFactory(identifier,'UCS:bios'),
+                led: dataFactory(identifier,'UCS:locator-led'),
+                skuInfo: Promise.try(function(){
+                            return node.sku === undefined ? "": node.sku;
+                        }),
+                chassis: Promise.map(node.relations,function(val){
+                            if(val.relationType === 'enclosedBy'){
+                                return val.targets;
+                            }
+                        }),
```

Comment:
If node has relations other than enclosedBy, what the Promise.map() will return?  I think the "chassis" item will be something like [nodeid, undefined,...]. Will it impact the following render process?

Reply:
I have tested it. It will not impact rendor process